Containerfile

```diff
@@ -37,14 +33,18 @@ LABEL org.opencontainers.image.authors="https://issues.redhat.com/browse/THREESC
       # org.opencontainers.image.created=""
 
 ARG PORTA_IMAGE=quay.io/3scale/porta:nightly
-COPY --from=builder /home/rpms /tmp/rpms
-COPY --from=builder /home/searchd /
-COPY --from=$PORTA_IMAGE /opt/system/config/standalone.sphinx.conf "/etc/sphinx/system.sphinx.conf"
-RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release && \
-    rpm -iv --excludedocs /tmp/rpms/* && \
-    rm -rf /tmp/rpms && \
-    mkdir -p /var/lib/searchd /var/run/sphinx && \
-    chmod 770 /var/lib/searchd /var/run/sphinx
+COPY --from=builder /tmp/manticore_uselessly_long_path_to_prevent_rpm_build_issues/build/*.rpm /tmp/rpms/
+COPY --from=$PORTA_IMAGE /opt/system/config/standalone.sphinx.conf "/etc/manticoresearch/manticore.conf"
+ENV MANTICORE_RPMS="manticore-converter* manticore-icudata* manticore-common* manticore-server-core* manticore-server*"
```

Comment:
Why not just `manticore-*` ? or would it contain something that we don't need?

Reply:
Yes, some RPMs have deps that are not satisfied. And we don't really need them.